src/components/Gallery/ImageGallery.css

```diff
@@ -14,6 +14,9 @@
   margin-top: 32px;
   margin-bottom: 32px;
   overflow: hidden;
+  overflow-y: scroll;
+  padding: 0px 40px;
+  max-height: 600px;
```

Comment:
Allow the gallery to be scrollable but don't allow the hight to be taller than the space allotted for the grid.

Reply:
I wonder if notes like these should live in the codebase as comments? That way they are both apparent to the reviewer as well as future maintainers

Reply:
I worry that doing so increases work for a maintainer should the commemt become outdated. But happy to add it if we feel it's unlikely to become stale.